language/Portuguese_Brazil/strings.po

```diff
@@ -426,19 +431,19 @@ msgstr "Opções de Performance"
 
 #: src/frontend/mame/ui/submenu.cpp:52
 msgid "Auto frame skip"
-msgstr "Salto de quadro Automático"
+msgstr "Auto frameskip"
 
 #: src/frontend/mame/ui/submenu.cpp:53
 msgid "Frame skip"
-msgstr "Salto de quadro"
+msgstr "Frame skip"
```

Comment:
Any specific reason for using untranslated terms here?  Is “frameskip” a common loanword in Brazilian Portuguese?  Also, if used as a loanword, would it be better to be consistent with use of “frameskip” (no space) vs “frame skip” (with space)?

Reply:
In brazilian sites dedicated to emulation certain words are said in english instead of portuguese, frameskip and save states are such examples, that's why i left them like that. ;)